packages/chevrotain/package.json

```diff
@@ -53,15 +54,16 @@
         "quick-build": "tsc && yarn run bundle",
         "---------- BUILD STEPS --------": "",
         "clean": "shx rm -rf coverage dev lib",
-        "clean:es6": "shx rm -rf es6",
+        "clean:esm": "shx rm -rf esm",
         "lint": "tslint --project tsconfig.json",
-        "lint:es6": "tslint --project tsconfig.es6.json",
+        "lint:esm": "tslint --project tsconfig.esm.json",
```

Comment:
why do we need a separate linter for ESM?

Reply:
They use different compilerOptions so I think the lint result may differ.

Reply:
Still seems strange to me as the linting is about the structure of the **source** code.
If we want to avoid certain patterns, e.g using module.exports we should setup a lint rule for that on the tslint level.
If we want to ensure the generated artifacts work we have unit tests for this purpose (with ~100% coverage)

Reply:
Because tslint also checks type information when specifying `--project`, which may differ with `compilerOptions`.

> https://palantir.github.io/tslint/usage/cli/
> -p, --project:
>     The path or directory containing a tsconfig.json file that will be
>     used to determine which files will be linted. This flag also enables
>     rules that require the type checker.

Reply:
But would we care if anything esm + tslint fails if the **runtime** (unit tests) + compilation (tsc) does not fail?

Reply:
I think you are right, separate linting may be redundant. Two tsconfig files only differ in target options and other compiler options are same. Thus if esm+linting fails, cjs+linting should also fail.